File: frontend/js/chat/models/OnlineUsers.js
export default class OnlineUsers {

    constructor(count, type="online_users"){
        this.count = count;
        this.type = type;
    }

    static fromData(data){
        return new OnlineUsers(data.count, data.type);
    }
    
    toData(){
        return {
            count: this.count,
            type: this.type
        }
    }

    getOnlineUsers(){
        return this.count;
    }

    getType(){
        return this.type;
    }  
}

File: backend/models/online_users.py
from .packet import Packet

class OnlineUsers(Packet):
    def __init__(self, count):
        self.count = count

    def to_data(self):
        return {
            "type": "online_users",
            "count": self.count
        }

    @staticmethod
    def from_data(data):
        return super().from_data(data)